components/calendar-notepad.tsx

```
@@ -157,13 +213,18 @@ export function CalendarNotepad({ chatId }: CalendarNotepadProps) {
                   <p className="text-xs text-muted-foreground mb-1">
                     {new Date(note.createdAt).toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' })}
                   </p>
-                  <p className="text-sm whitespace-pre-wrap break-words">{note.content}</p>
+                  <p className="text-sm whitespace-pre-wrap break-words">{renderContent(note.content)}</p>
                 </div>
                 {note.locationTags && (
                   <button onClick={() => handleFlyTo(note.locationTags)} className="text-muted-foreground hover:text-foreground ml-2">
                     <MapPin className="h-5 w-5" />
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)